### DISCUSSION TITLE
Warning when prayer level is low

### DISCUSSION THREAD

**Discussion by data-monkeyy** [runelite/runelite#18005] [Ideas]
Hey, got an idea for a plugin: would be nice if you can set a threshold for prayer points below which the screen flashes a light red overlay.

Is this something the community would want? I'm going to develop it for myself but happy to share it if people want it too :)

**Comment by Felanbird** [comment]
The hub plugin [health notifications](https://runelite.net/plugin-hub/show/health-notifications) also does prayer I think, but it might be blue instead of red (because health is red)

**Comment by YvesW** [comment]
Additionally, rl core also does this (idle notifier).

**Comment by data-monkeyy** [comment]
Ah thanks guys yes looks like this feature already exists
